Changes

```diff
@@ -614,6 +614,9 @@ Working version
   types.
   (Chris Casinghino, review by Gabriel Scherer)
 
+- #12056, #12057, #??: Random failure in Marshal.from_{string,bytes}
+  (Damien Doligez, review by ??)
```

Comment:
I'll let to Changelog police tell you if you need a separate entry for these or if they should be merged with #12006 .

Reply:
police heree: it's as Damien prefers.